src/components/Attachments/AttachmentView/AttachmentViewImage/index.tsx

```diff
@@ -30,7 +43,7 @@ function AttachmentViewImage({url, file, isAuthTokenRequired, isFocused, loadCom
             disabled={loadComplete}
             style={[styles.flex1, styles.flexRow, styles.alignSelfStretch]}
             accessibilityRole={CONST.ACCESSIBILITY_ROLE.IMAGEBUTTON}
-            accessibilityLabel={file.name || translate('attachmentView.unknownFilename')}
+            accessibilityLabel={file?.name ?? translate('attachmentView.unknownFilename')}
```

Comment:
```suggestion
            accessibilityLabel={file?.name || translate('attachmentView.unknownFilename')}
```
If name is an empty string we want to use the fallback string